native/explorer/src/series.rs

```diff
@@ -456,6 +456,34 @@ pub fn s_fill_missing_with_bin(data: ExSeries, strategy: &str) -> Result<ExSerie
     Ok(ExSeries::new(s))
 }
 
+#[rustler::nif(schedule = "DirtyCpu")]
+pub fn s_fill_missing_with_date(
+    data: ExSeries,
+    strategy: ExDate,
```

Comment:
It's nitpick, but I would name this "`date: ExDate`" instead of `strategy`.
The same idea for the `s_fill_missing_with_datetime`. WDYT?

Reply:
Yes, I agree, but other functions follow this naming pattern, such as: s_fill_missing_with_atom, s_fill_missing_with_int, s_fill_missing_with_float, s_fill_missing_with_bin, and s_fill_missing_with_boolean. WDYT?

Reply:
I'm more inclined to rename them to be more closed to the data type when it's possible. I guess where we are using `fill_null_with_values` we could rename. WDYT?